wiki/People/The_Team/Accounts_management_team/en.md

```diff
@@ -0,0 +1,31 @@
+# Account management team
```

Comment:
I am hesitant to do that, because it's a generalization and not a proper noun, so sentence case fits here, even though it's off-putting compared to the rest of "The Team" articles